hack/kind/kind.sh

```diff
@@ -13,10 +13,13 @@
 
 # Exit immediately for non zero status
 set -e
-# Print commands
-set -x
 
-KIND_LOGGING=${KIND_LOGGING:-"-v 1"}
+if [ -n "${KIND_LOGGING}" ]; then
+  IFS=' ' read -r -a log_lvl <<< "${KIND_LOGGING}"
+else
+  log_lvl=("-v" "1")
+fi
+
```

Comment:
I think it would be simpler this way:

```suggestion
KIND_LOG_LEVEL=${KIND_LOG_LEVEL:-"1"}
log_lvl=("-v" "$KIND_LOG_LEVEL")
```

WDYT?

Reply:
👍  good call